script/deploy/facets/UpdateCoreFacets.s.sol

```diff
@@ -110,6 +114,14 @@ contract DeployScript is UpdateScriptBase {
             buildInitialCut(selectors, peripheryRgs);
         }
 
+        // GasZipFacet
+        selectors = getSelectors("GasZipFacet", exclude);
+        if (loupeExists) {
+            buildDiamondCut(selectors, gasZipAddress);
+        } else {
+            buildInitialCut(selectors, gasZipAddress);
+        }
+
```

Comment:
in this file there is a lot of repetitive code, following DRY principle what do you think about reducing logic into single reusable function? I believe that in this file we can have a function with list of names and for the next core facet just add new element. Or what could be even better, core facet could be taken from global.json to keep list of core facets in one place

Reply:
yes, great suggestion. Very good point. Didnt think about that at all.
Updated.